### PR TITLE
Fix: function description in test_module.py

### PR DESCRIPTION
**Pull request by Abhijeet1520**
Fix: function description in test_module.py

fix test_two_days_later_with_day function's assertEqual message in test_module.py

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions!

Please note that our workflow requires at least one additional approval before this can be merged.

**Review by moT01 — APPROVED** [review]
LGTM @Abhijeet1520 🎉

**Pull request merged**
